content/en/docs/contribute/localization.md

```diff
@@ -68,50 +72,93 @@ contentDir = "content/de"
 weight = 3
 ```
 
-When assigning a `weight` parameter, see which of the current languages has the highest weight and add 1 to that value.
+When assigning a `weight` parameter for your block, find the language block with the highest weight and add 1 to that value.
+
+For more information about Hugo's multilingual support, see "[Multilingual Mode](https://gohugo.io/content-management/multilingual/)".
+
+### Add a new localization directory
 
-Now add a language-specific subdirectory to the [`content`](https://github.com/kubernetes/website/tree/master/content) folder. The two-letter code for German is `de`, so add a `content/de` directory:
+Add a language-specific subdirectory to the [`content`](https://github.com/kubernetes/website/tree/master/content) folder in the repository. For example, the two-letter code for German is `de`:
 
 ```shell
 mkdir content/de
 ```
 
+### Add a localized README 
+
+To guide other localization contributors, add a new [`README-**.md`](https://help.github.com/articles/about-readmes/) to the top level of k/website, where `**` is the two-letter language code. For example, a German README file would be `README-de.md`. 
```

Comment:
@cstoku ☝️ The addition of language-specific `README-*` files is why want to verify the regex for `"README*.md"`.